fe/fe-core/src/main/java/com/starrocks/credential/hdfs/StrictHDFSCloudConfigurationProvider.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.credential.hdfs;
+
+import com.starrocks.credential.CloudConfiguration;
+
+import java.util.Map;
+
+import static com.starrocks.credential.CloudConfigurationConstants.HADOOP_KERBEROS_KEYTAB;
```

Comment:
may be can just use `com.starrocks.credential.CloudConfigurationConstants;` to save these long list of static import?